### ISSUE TITLE
Bug report: only some operation tests are being run

### ISSUE THREAD

**Issue by d98762625** [gchq/CyberChef#465]
Bug report:

### Summary

Not all operation tests are run with `npm test` or `grunt test`, plus a varying number of tests are run each time.

### Example
```
git clone git@github.com:gchq/CyberChef.git
cd CyberChef
npm install
npm test
```

First run: last test is ✔️️ HMAC: RFC4231 Test Case 4 SHA-256
Second run: last test is ✔️️ HMAC: RFC4231 Test Case 3 SHA-224
Third run: last test is ✔️️ HMAC: RFC4231 Test Case 3 SHA-256

If you stick `console.log`s in the test suite, fewer results are logged - kind of like it bails and 'completes' after a certain amount of time

```
grunt --version
grunt-cli v1.2.0
grunt v1.0.3
```

**Comment by d98762625** [comment]
Update from grunt-cli `v1.2.0` to `v1.3.2` yields similar result

**Comment by d98762625** [comment]
I'm not sure If I was just being stupid earlier or if I accidentally fixed this, but it's gone away in the [node-lib](https://github.com/d98762625/CyberChef/tree/node-lib) branch of my fork, which will be merged when #291 goes through
